Mathlib/RingTheory/IsAdjoinRoot.lean

```diff
@@ -635,8 +614,10 @@ end IsAdjoinRoot
 
 namespace IsAdjoinRootMonic
 
+variable (h : IsAdjoinRootMonic S f)
+
 theorem minpoly_eq [IsDomain R] [IsDomain S] [NoZeroSMulDivisors R S] [IsIntegrallyClosed R]
```

Comment:
One thing that's a bit weird about this change is that it puts `h` before the typeclass arguments, but we normally prefer these to come first.